packages/python-packages/api-stub-generator/ci.yml

```diff
@@ -0,0 +1,45 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - packages/python-packages/api-stub-generator
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - packages/python-packages/api-stub-generator
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tool-python.yml
+  parameters:
+    PythonVersion: '3.10.2'
+    PackagePath: 'packages/python-packages/api-stub-generator'
+    FeedName: 'public/azure-sdk-api-review-tools'
```

Comment:
Still curious if this is the best feed? We are trying to consume fewer feeds so I would suggest using the azure-sdk-for-python feed unless there is a reason not to.

Reply:
I was thinking of grouping a separate feed for apiview tools. But I am fine to update this to azure-sdk-for-python feedas well. @scbedd Let me know what you think.

Reply:
I don't mind! I'll update that before merging.

Reply:
[Build Targeting Python Feed](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1483829&view=results)